README.md

```diff
@@ -1,60 +1,48 @@
-# SnarkyJS
+# SnarkyJS &nbsp; [![npm version](https://img.shields.io/npm/v/snarkyjs.svg?style=flat)](https://www.npmjs.com/package/snarkyjs) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](#contributing)
 
 To write a zkApp, we recommend using the [zkApp CLI](https://github.com/o1-labs/zkapp-cli), which makes writing a zkApp easy by including SnarkyJS & providing project scaffolding, a test framework, and formatting.
 
 SnarkyJS's documentation is available [here](https://docs.minaprotocol.com/en/zkapps/snarkyjs-reference).
 
 See [CHANGELOG.md](https://github.com/o1-labs/snarkyjs/blob/main/CHANGELOG.md) for a list of changes between versions.
 
-## Run examples in Node
+## Community packages
 
-```sh
-npm install
-npm run build
+> An initial list of community-maintained packages will be put together soon. If you want your package to appear on it, see [the next section](#contributing)
 
-./run src/examples/api_exploration.ts
-```
-
-## Build and run web version
-
-```sh
-npm install
-npm run build:web
-npm run serve:web
-```
-
-Then go to `http://localhost:8000/`
-
-## Run tests
+## Contributing
 
-- Unit tests
+We appreciate any community contributions to SnarkyJS! There are two ways to contribute:
 
-  ```sh
-  npm run test
-  npm run test:unit
-  ```
+1. Maintain your own package, which people can install in addition to SnarkyJS
+2. Contribute to this repo directly
```

Comment:
Instead of writing this detail twice, I've added a link to the specific section that does explain this "do RFC first" step

Reply:
Not sure here, but I'll go ahead and resolve this conversation since the comment was addressed, although not literally... teach me if I need to leave these types of discretionary comments open, ty!